app/views/compute_resources_vms/form/vmware/_volume.html.erb

```diff
@@ -11,7 +11,7 @@
 <% else %>
   <%= text_f f, :datastore, :class => "span5", :label => _("Data store"), :label_size => "col-md-2", :disabled => true %>
 <% end %>
-<%= text_f f, :name, :class => "col-md-2", :label => _("Name"), :label_size => "col-md-2", :disabled => !new_host, :readonly => true %>
+<%= text_f f, :name, :class => "col-md-2", :label => _("Name"), :label_size => "col-md-2", :readonly => true %>
```

Comment:
That only disables editing, but the value still gets sent to the server in this case. if `:disabled => true` then the value isn't sent to the server at all. This might still be ok in some cases, just not sure if this is one of them.